README.md

```diff
@@ -150,6 +150,24 @@ When the code is ready, submit a Pull Request.
 - Prefer code that is readable over one that is "clever".
 - When writing a Git commit message, follow these [guidelines](https://chris.beams.io/posts/git-commit/).
 
+## Releasing
+
+- [Set up your development environment](#Getting-Started-With-Development).
+- Change version in `presto/__init__.py`.
+- Commit and create an annotated tag (`git tag -m '' current_version`)
+- Run the following:
+  ```bash
```

Comment:
nit: a script would be nice to have. Though I will not push on that.

Reply:
follow-up improvement